10_kata.py

```diff
@@ -0,0 +1,8 @@
+# https://www.codewars.com/kata/5287e858c6b5a9678200083c/train/python
+
+def narcissistic(value):
+
+    digits = list(str(value))
```

Comment:
The transformation to list is not necessary. You can loop over a string the same way you do over a list. The for loop goes over every single character in a string. For example if the `digits = str(value)` is equal to `'3215'`, then:
`for digit in digits:
    print(digit)`
Will give you a single digit in every iteration: 3, 2, 1, 5.